File: src/main/java/me/nasukhov/TukitaLearner/LearnerApp.java
package me.nasukhov.TukitaLearner;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.WebApplicationType;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class LearnerApp {
    public static void main(String[] args) {
        var app = new SpringApplication(LearnerApp.class);
        app.setWebApplicationType(WebApplicationType.NONE);
        app.run(args);
    }
}

File: src/main/kotlin/me/nasukhov/tukitalearner/LearnerApp.kt
package me.nasukhov.tukitalearner

import org.springframework.boot.WebApplicationType
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.builder.SpringApplicationBuilder

@SpringBootApplication
class LearnerApp

@Suppress("SpreadOperator")
fun main(args: Array<String>) {
    SpringApplicationBuilder(LearnerApp::class.java)
        .web(WebApplicationType.NONE)
        .run(*args)
}
